packages/wonder-stuff-sentry/src/kind-sentry-error.js

```diff
@@ -0,0 +1,177 @@
+// @flow
+import {KindError, Errors} from "@khanacademy/wonder-stuff-core";
+import type {Metadata} from "@khanacademy/wonder-stuff-core";
+import {sentryDataReducer} from "./sentry-data-reducer.js";
+import type {SentryData} from "./types.js";
+
+const EmptySentryData: $ReadOnly<SentryData> = Object.freeze({
+    tags: {},
+    contexts: {},
+    fingerprint: [],
+});
+
+/**
+ * Options for constructing a `KindError`.
+ */
+type Options = {|
```

Comment:
Do we need to duplicate this object here? Or can we just spread `kind-errors`'s into this one and add the Sentry data?

Reply:
I kept them separate for now because it's not clear what options will get shared versus replaced, etc. I expect some later PR will do some consolidation as needed.